tests/test_back.js

```diff
@@ -322,23 +322,44 @@ tap.test('nockBack record tests', function (nw) {
   //Adding this test because there was an issue when not calling
   //nock.activate() after calling nock.restore()
   nw.test('it can record twice', {skip: process.env.AIRPLANE}, function (t) {
+
```

Comment:
nit: remove the empty line on top of a function

Reply:
I added it deliberately. Found it to improve readability for me. I'll remove it.

Reply:
I might just be too used to the standard style :) thanks